src/rgw/rgw_rados.cc

```diff
@@ -318,6 +318,11 @@ const string RGWZoneGroup::get_default_oid(bool old_region_format)
     return cct->_conf->rgw_default_region_info_oid;
   }
 
+  //no default realm exist
+  if (realm_id.empty()){
+    return zonegroup_names_oid_prefix + default_zonegroup_name;
+  }
+
```

Comment:
note that `RGWSystemMetaObj::read_default()` will try to decode this object into a `RGWDefaultSystemMetaObjInfo` - but the name object at `default_zonegroup_name` would contain a `RGWNameToId` for `RGWSystemMetaObj::read_id()` instead